README.md

```diff
@@ -350,10 +350,10 @@ contractual rights that later can lead to owning stock.
 These forms include restricted stock, stock options, and RSUs, each of which we’ll
 [describe in detail](#how-equity-is-granted).
 
-☝️ The word “equity” has
+☝️ The word *equity* has
 [several technical meanings](https://www.investopedia.com/terms/e/equity.asp) in accounting
-and other financial contexts, but when we talk about equity compensation, it refers to
-ownership in a company you work for.
+and other financial contexts, but *equity compensation* refers to ownership in a company
```

Comment:
Nit: We already defined "equity" so what we want to say is, every occurrence of that single word in this guide is in the context of compensation. Not the same as saying what it means only for "equity compensation".

So if we're editing this, a better phrasing would be:

but when equity is discussed in the context of compensation, it refers to ownership in a company you work for.

Reply:
Will correct in next copy edit!

Reply:
Thanks!